data/users.js

```diff
@@ -0,0 +1,74 @@
+const users = [
+    {
+        'id': '9b3fafe2-76b1-4a8b-b668-4177e1e820f5',
+        'login': 'fitl_541',
+        'password': 'qweee4ld1',
+        'age': 27,
+        'isdeleted': true
+    },
+    {
+        'id': '77446a9d-0e46-43da-8f1b-60614432a318',
+        'login': 'liy_95dfgdf4d_d',
+        'password': 'voi459dfgdfddd',
+        'age': 26,
+        'isdeleted': false
+    },
+    {
+        'id': '34e52832-5769-42f0-bf08-38d675ea6e52',
+        'login': 'fitsddddddddddddddddddddddddddddddddddddl4',
+        'password': 'qweee45',
+        'age': 27,
+        'isdeleted': false
+    },
+    {
+        'id': '7240630e-d54e-4311-a90e-0eed23c5d0b7',
+        'login': 'chiron',
+        'password': 'coxreikk',
+        'age': 23,
+        'isdeleted': false
+    },
+    {
+        'id': 'a0ba7091-c703-4861-adac-0873cbaa04d4',
+        'login': 'versio345',
+        'password': 'sdfjjj5456',
+        'age': 23,
+        'isdeleted': false
+    },
+    {
+        'id': '8a3e7f0b-824a-4a5f-bcea-0128fb829f97',
+        'login': 'nodemon',
+        'password': 'nodejs',

```

Comment:
It's better not to include sensitive data (password) and internal data (isdeleted) to response